multi-cluster-ingress/multi-cluster-blue-green-cluster/README.md

```diff
@@ -0,0 +1,355 @@
+        mci-6rsucs-8080-multi-cluster-demo-default-backend
+        Network Endpoint Groups:
+        zones/us-west1-a/networkEndpointGroups/k8s1-3efcccf3-multi-cluste-mci-default-backend-svc--80-b2c88574
+        zones/us-west1-b/networkEndpointGroups/k8s1-6c1ae7d4-multi-cluste-mci-default-backend-svc--80-f8b91776
+        Target Proxies:
+        mci-6rsucs-multi-cluster-demo-foobar-ingress
+        URL Map:  mci-6rsucs-multi-cluster-demo-foobar-ingress
+    VIP:        34.120.46.9
+    Events:
+    Type     Reason  Age                  From                              Message
+    ----     ------  ----                 ----                              -------
+    Normal   ADD     12m                  multi-cluster-ingress-controller  multi-cluster-demo/foobar-ingress
+    Warning  SYNC    5m31s (x2 over 10m)  multi-cluster-ingress-controller  Error AVMBR102: MultiClusterService multi-cluster-demo/default-backend not found.
+    Normal   UPDATE  98s (x2 over 12m)    multi-cluster-ingress-controller  multi-cluster-demo/foobar-ingress
+    ```
+
+    ```bash
+    # capture the IP address for the MCI resource
+    $ export MCI_ENDPOINT=$(kubectl --context=gke-1 get mci -n multi-cluster-demo -o yaml | grep "VIP" | awk 'END{ print $2}')
+    ```
+
+7. Now use the IP address from the MCI output to reach the load balancer. Running it several times should reflect that traffic is being load-balanced between `gke-1` and `gke-3`. We use `jq` to filter the output to make it easier to read but you could drop the `jq` portion of the command to see the full output.
+
+    ```bash
+    # Hitting the default backend attempt #1
+    $ curl -s ${MCI_ENDPOINT} | jq -r '.zone, .cluster_name, .pod_name'
+    us-west1-a
+    gke-1
+    default-backend-85798bc9b5-bnqhr
+
+    # Hitting the default backend attempt #2
+    $ curl -s ${MCI_ENDPOINT} | jq -r '.zone, .cluster_name, .pod_name'
+    us-west1-b
+    gke-3
+    default-backend-85798bc9b5-wpb59
+    ```
+
+
+8. Now let's demonstrate how to take one of the clusters (in this case, `gke-1`) temporarily out of service. Start by sending requests to the MCI endpoint.
+
+    ```bash
+    $ while true; do curl -s ${MCI_ENDPOINT} | jq -c '{cluster: .cluster_name, pod: .pod_name}'; sleep 2; done
+
+    {"cluster":"gke-3","pod":"default-backend-85798bc9b5-wpb59"}
+    {"cluster":"gke-1","pod":"default-backend-85798bc9b5-bnqhr"}
+    {"cluster":"gke-3","pod":"default-backend-85798bc9b5-wpb59"}
+    {"cluster":"gke-1","pod":"default-backend-85798bc9b5-bnqhr"}
+    {"cluster":"gke-3","pod":"default-backend-85798bc9b5-wpb59"}
+    {"cluster":"gke-1","pod":"default-backend-85798bc9b5-bnqhr"}
+    ...
+    ```
+
+**Note:** This failover process will take several minutes to take effect.
+
+9. Open up a second shell to remove `gke-1` from the MultiClusterService via `patch`.
+
+    ```bash
+    $ kubectl patch MultiClusterService default-backend -n multi-cluster-demo --type merge --patch "$(cat ./patch.yaml)"
```

Comment:
I would make a reference back to the MCS v2 with gke-1 removed from the MCS that this patch.yaml applies. This is the most critical step in the guide so this will help the user understand exactly what's being done here.